pkg/admission/customresourcevalidation/customresourcevalidationregistration/cr_validation_registration.go

```diff
@@ -34,4 +36,7 @@ func RegisterCustomResourceValidation(plugins *admission.Plugins) {
 	project.Register(plugins)
 	config.Register(plugins)
 	scheduler.Register(plugins)
+
+	// This plugin validates the quota.openshift.io/v1 ClusterResourceQuota resources.
```

Comment:
indicate that this is only allowed because it is required to get a running control plane operator.